### PR TITLE
build(docker): Two compose files for arm and amd

### PR DESCRIPTION
**Pull request by prashant-shahi**
- remove `--env-file` usage from install script and contributing docs
- use `./data` as `data` volume binding for docker
- remove `.gitkeep` files from `data` folder
- pins image tags in docker-compose
- reverts #628

Signed-off-by: Prashant Shahi <prashant@signoz.io>

### CODE REVIEW

**Comment by prashant-shahi** [comment]
yeah.. will do

**Comment by pranay01** [comment]
@prashant-shahi Can you also create a PR on signoz.io repo for updating docs based on this change

**Comment by prashant-shahi** [comment]
Docs updated: https://github.com/SigNoz/signoz.io/pull/161

**Review by ankitnayan — CHANGES_REQUESTED** [review]

File: deploy/docker/clickhouse-setup/docker-compose.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: "2.4"
 
 services:
   clickhouse:
-      image: ${clickhouse_image}
+      image: yandex/clickhouse-server:21.12.3.32
```
[the remark this answers is not in the document]
Comment (prashant-shahi) [reply deploy/docker/clickhouse-setup/docker-compose.yaml]: I choose `21.12` as I found a matching major-minor version clickhouse image tag for AMD by yandex and ARM by altinity.
Which I wasn't able to find `21.10`.
If we decide to have a matching version then, we could either downgrade to `21.8` or upgrade to `21.12`.
So, I went with the latter.

[the remark this answers is not in the document]
Comment (ankitnayan) [reply deploy/docker/clickhouse-setup/docker-compose.yaml]: got it

**Review by ankitnayan — APPROVED** [review]

**Pull request merged**